### PR TITLE
app preview value lengths

### PR DESCRIPTION
**Pull request by jmtroth0**
https://dimagi-dev.atlassian.net/browse/HI-259

not exactly a bug, but for that one person that actually needs a big number, here ya go

@calellowitz

### CODE REVIEW

**Comment by jmtroth0** [comment]
(comments were outdated)

**Comment by jmtroth0** [comment]
who else can i ping on formplayer stuff? @orangejenny

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js [hunk 1]
```diff
@@ -262,9 +262,15 @@ PhoneEntry.prototype.constructor = FreeTextEntry;
 function FloatEntry(question, options) {
     IntEntry.call(this, question, options);
     this.templateType = 'str';
+    this.lengthLimit = options.lengthLimit || 15;
+    var valueLimit = options.valueLimit || +("9".repeat(14));
 
     this.getErrorMessage = function (rawAnswer) {
-        return (isNaN(+rawAnswer) ? "Not a valid number" : null);
+        if (isNaN(+rawAnswer))
+            return "Not a valid number";
```
Comment (orangejenny) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: Not related to this PR, but shouldn't these be translated?

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Not blocking, but it'd be an improvement to move the magic numbers into constants.js

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js (hunk 1 above)
Comment (jmtroth0) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: Hm, probably. I think there are a bunch of things in formplayer of which that is true

**Review by orangejenny — APPROVED** [review]
Nice, thanks for updates.

**Pull request merged**